app/config/AppConfig.scala

```diff
@@ -62,7 +48,19 @@ class AppConfigImpl @Inject() (val servicesConfig: ServicesConfig, configuration
 
   def apiStatus(version: String): String = servicesConfig.getString(s"api.$version.status")
 
-  def featureSwitch: Option[Configuration] = configuration.getOptional[Configuration](s"feature-switch")
+  def featureSwitch: Option[Configuration] = configuration.getOptional[Configuration]("feature-switch")
 
   lazy val endpointsEnabled: Boolean = servicesConfig.getBoolean("api-definitions.endpoints.enabled")
+
+  // HIP
+  private lazy val hipUrl: String     = servicesConfig.baseUrl("hip")
+  private lazy val baseHipUrl: String = s"$hipUrl/cir"
+
+  lazy val isHipEnabled: Boolean =
+    configuration.getOptional[Boolean]("microservice.services.hip.enabled").getOrElse(false)
```

Comment:
in application.conf it's hip.hipEnabled. I'd change the application.conf file to this microservice.services.hip.enabled